src/server.cpp

```diff
@@ -2315,6 +2316,7 @@ void Server::fillMediaCache()
 				".pcx", ".ppm", ".psd", ".wal", ".rgb",
 				".ogg",
 				".x", ".b3d", ".md2", ".obj",
+				".tr",
```

Comment:
might be a good idea to add a comment indicating that this is our custom translation text format